lib/deqm_test_kit/bulk_import.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require_relative '../utils/bulk_import_utils'
+module DEQMTestKit
+  # BulkImport test group ensure the fhir server can accept bulk data import requests
+  class BulkImport < Inferno::TestGroup
+    id 'bulk_import'
+    title 'Bulk Import'
+    description 'Ensure the fhir server can accept bulk data import requests'
+
+    input :measure_id
+    custom_headers = { 'X-Provenance': '{"resourceType": "Provenance"}', prefer: 'respond-async' }
+    params = {
```

Comment:
Could potentially use FHIR::Parameters.new() to build this for a little bit of easier maintenance and property verification in code.

Reply:
Looks like this is still relevant. Unless there was an issue with this approach?

Reply:
Yeah when I pushed the change to this, I broke the spec tests even after following the example in the submit data tests, I'm not sure if it was the way I was doing it or it was just the way the data was formatted